frontend/src/metabase/embedding-sdk/components/SimpleDataPicker/SimpleDataPicker.tsx

```diff
@@ -1,108 +1,94 @@
-import { useState } from "react";
-import { t } from "ttag";
+import { useDisclosure } from "@mantine/hooks";
```

Comment:
this file diff looks confusing, but basically `SimpleDataPicker` was a view component before.

Now it's renamed to `SimpleDataPickerView` and `SimpleDataPicker` is a container component.

Reply:
I started using split-view diffs these days, makes reviewing these types of diffs so much easier 😄